packages/flutter/test/material/app_bar_test.dart

```diff
@@ -2339,6 +2341,39 @@ void main() {
       expect(tester.getSize(findAppBarMaterial()).height, kToolbarHeight);
     });
 
+    testWidgets('backgroundColor animation', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        buildAppBar(contentHeight: 1200.0, surfaceTintColor: Colors.transparent),
```

Comment:
`surfaceTintColor` has been deprecated in [specs](https://m3.material.io/components/top-app-bar/specs#1adcfb52-bd43-4620-aef1-ad6d60c1f7f5), even though we haven't cleaned it up here. So I think it would be better if we don't introduce more use case like this. Instead of changing `surfaceTintColor` to transparent, I think assigning `scrolledUnderElevation: 0` in appbar will also work for this test.

Reply:
Sure, let me push the change.